docs: add key terms to use case intros/tutorial and what is dvc? docs [SEO]

jeremydesroches opened the pull request:
Based on existing search results, I expanded the use case intro docs with data and model versioning references. The tutorial received similar changes — pending review with @jorgeorpinel.

Merged PR #1805 (meant to separate commits but I messed it up 😒).

**/docs/user-guide/what-is-dvc.md**
Add explicit "machine learning" references including "version machine learning experiments"

**/docs/use-cases/index.md**
Add "data science use cases", "tools" and "best practices"

**/docs/use-cases/versioning-data-and-model-files/index.md**
Add "data and model versioning", "versioning (large) data files", and "model versions"

**/docs/use-cases/versioning-data-and-model-files/tutorial.md**
Add "ML model versions", "dataset versioning", "model and large dataset versioning", "machine learning models", "dataset and ML model versioning"

jorgeorpinel commented on content/docs/user-guide/what-is-dvc.md:
```diff
@@ -1,16 +1,18 @@
-(although it can work stand-alone). DVC reduces the gap between established
-engineering tool sets and data science needs, allowing users to take advantage
-of new [features](#core-features) while reusing existing skills and intuition.
+**Data Version Control** is a new type of data versioning, workflow and machine
+learning experiment management software that builds upon
```
> machine learning experiment management

Makes sense but doesn't even register as a term (https://trends.google.com/trends/explore?geo=US&q=machine%20learning%20experiment%20management,experiment%20management). Then again "experiment management software" doesn't register either... So not sure it's a problem. But is there another way to insert "machine learning" in this paragraph without making it hard to read? Or maybe just use "ML" here and insert the full term elsewhere in the p/doc?

jeremydesroches replied:
Was after specific term "[version machine learning experiments](https://www.google.com/search?client=safari&rls=en&q=version+machine+learning+experiments&ie=UTF-8&oe=UTF-8)", which is awkward to insert. I like the idea to use "ML" here... there's already a lot going on in that sentence. 

In a lot of cases a term might appear in search console or related terms, and I'm looking to expand where DVC.org has impressions but low/zero clicks. Many are domain-specific and very low volume (relative to Internet scale) so they might not appear in Google trends at all.

jorgeorpinel replied:
> Was after specific term "version machine learning experiments"

I see. Let's find another place to add the term in this doc? Feel free to add sentences, paragraphs, or even sections if needed.

jeremydesroches replied:
OK, added this in a rewrite of the other paragraph.

jorgeorpinel commented on content/docs/user-guide/what-is-dvc.md:
```diff
@@ -1,16 +1,18 @@
+users to take advantage of new [features](#core-features) while reusing existing
+skills and intuition.
 
-![](/img/reproducibility.png) _DVC codifies data and ML experiments_
+![](/img/reproducibility.png) _DVC codifies data and machine learning
+experiments_
```
I wonder whether having "ML" along with "machine learning" in the same doc also helps, or should abbreviated terms be avoided?

jeremydesroches replied:
> I wonder whether having "ML" along with "machine learning" in the same doc also helps, or should abbreviated terms be avoided?

Yes, it does help. Ideally we have both, including the combined usage. For example, "ML experiments" and "machine learning experiments".

jorgeorpinel replied:
OK! So I think this change can be revered? Since the full term will be added somewhere before this image (probably). An img caption seems like a good place for the abbr form.

jeremydesroches replied:
Agree abbr better for caption and added full usage in rewritten paragraph.

jorgeorpinel commented on content/docs/user-guide/what-is-dvc.md:
```diff
@@ -1,16 +1,18 @@
+experiments_
 
-Data science experiment sharing and collaboration can be done through a regular
-Git flow (commits, branching, pull requests, etc.), the same way it works for
-software engineers.
+Using DVC, data scientists can apply a regular Git flow to ML project sharing
+and collaboration (commits, branching, pull requests, etc.), the same way it
+works for software engineers.
```
I see "ML" is used here. What are the guidelines to decide whether to use the full term or an abbreviation?

Also, this change kind of modifies the meaning of the text. We were previously talking about sharing data science experiments, now we're talking about data scientists sharing ML projects. Can we avoid semantic changes if the goal of the PR is mainly to insert a term? Unless you want to suggest a better writing of course, but not sure that was your intention.

jeremydesroches replied:
> I see "ML" is used here. What are the guidelines to decide whether to use the full term or an abbreviation?

Clarity can remain the priority. We just want both abbreviated and unabbreviated forms on a document level, including the combined usage variants we're pursuing for SEO.
 
> Can we avoid semantic changes if the goal of the PR is mainly to insert a term? 

Yes, of course. I was trying to get "ML" as close to "flow" as possible to address the competitive term "MLflow". This is a fairly aggressive approach — we don't need to go this far. For this initial PR I wanted added examples both less and more aggressive so we could decide on what is acceptable and what is "too far". 😉

jorgeorpinel replied:
> I was trying to get "ML" as close to "flow" as possible to address the competitive term "MLflow".

Oh, interesting. In that case we could consider this change. What could that achieve in terms of SEO? Cc @shcheklein 

BTW you just removed "ML" now so we don't get the new term benefit and lose the original meaning. I'd try reverting it first, and then rewrite it again 🙂

jeremydesroches replied:
OK, added the main term I'm looking to add in an expanded version of this paragraph. I think the addition fulfills the primary need for this doc so we can leave the ML + flow consideration for elsewhere.

jorgeorpinel commented:
Reviewed Whas is DVC? for now ☝️ 

BTW, you should have access to push branches to this repo @jeremydesroches so no need to use a fork going forward 🙂. Using branches directly on `origin` will trigger a review app that we can all see online, so it's preferred. Thanks

jorgeorpinel commented:
> push branches to this repo

No need to do it for this PR BTW, but if yo uhave any others that haven't been reviewed, those would be good to close and reopen from direct branches. Thanks

jeremydesroches commented:
> BTW, you should have access to push branches to this repo @jeremydesroches so no need to use a fork going forward 🙂. Using branches directly on `origin` will trigger a review app that we can all see online, so it's preferred. Thanks

Awesome. I will do that! Thanks @jorgeorpinel

jorgeorpinel commented on content/docs/user-guide/what-is-dvc.md:
```diff
@@ -1,16 +1,17 @@
-Git flow (commits, branching, pull requests, etc.), the same way it works for
-software engineers.
+The same way that software engineers use Git, data scientists can use DVC to
+apply a regular flow to project sharing and collaboration (commits, branching,
+pull requests, etc.). Using Git and DVC, data science and machine learning teams
+can version experiments, manage large datasets, and make projects reproducible.
```
I like it! You could even put flow and ML closer like this:
```suggestion
apply a regular flow to machine learning project collaboration and sharing (commits, branching,
pull requests, etc.). Using Git and DVC, data science teams
can version large datasets, manage experiments, and make projects reproducible.
```
> Needs formatting

Notice I also reordered some list items in a way I think makes a little more sense. But in terms of the right terms, looks pretty complete. Maybe even too much for a single paragraph haha.

jorgeorpinel replied:
Wait no... We still changed the meaning of the paragraph 🤔 

Let's keep the topic about _data science experiment collaboration and sharing_ please. If we need to talk about _regular flow to machine learning projects_ separately, let's use a separate sentence/paragraph. Thanks

jeremydesroches replied:
A bit confused with the feedback here. 

My initial version had "...a regular Git flow to ML project...". Your initial feedback was that changed the meaning, so I removed the "ML" there and expanded the paragraph to use the complete "machine learning" term and additional features (an additional intro to the features below). 

So my latest doesn't have "regular flow to machine learning projects" — it is in a separate sentence/paragraph. The added sentence is better in SEO terms than my original version with "ML + flow", so I left it out on purpose.

jorgeorpinel replied:
Yeah sorry I went back and forth on this one...

jorgeorpinel replied:
> Your initial feedback was that changed the meaning

Yes but it didn't refer to where/how to insert the "ML" term or to the extra info added (that's all good), but rather to avoid switching the subject from "data science experiments" to "flow to machine learning projects".

jorgeorpinel replied:
> separate sentence/paragraph. The added sentence is better in SEO terms

That's OK. I'm talking about the change in the 1st sentence.

jeremydesroches replied:
My mistake. I get you now... I wasn't looking at the original version. Updated to retain the original 1st sentence and add the new 2nd sentence.

jorgeorpinel replied:
Great. Thanks!

jorgeorpinel commented on content/docs/use-cases/index.md:
```diff
@@ -1,28 +1,28 @@
-that DVC can help with or improve. Our use cases are not written to be run
-end-to-end like tutorials. For more general, hands-on experience with DVC,
-please see our [Get Started](/doc/tutorials/get-started) instead.
+We provide short articles on common ML workflow and data science use cases that
```
Is this a copy edit? I do agree about "and" but why did we change "scenarios"? If there's no SEO reason let's revert that please:

```suggestion
We provide short articles on common ML workflow and data science scenarios that
```

jeremydesroches replied:
Yes there is an SEO motivation here: the search term is "data science use cases".

jorgeorpinel replied:
I see! Going fwd if you can make some notes in the PR file changes on terms each change is for, or a list of terms in the PR description at least, that would be helpful for reviews 😃

jeremydesroches replied:
👍 Definitely. That makes a lot of sense and I'll do that.

jorgeorpinel replied:
Does it matter that probably users looking for "data science use cases" are not looking for DVC use cases? I don't want to assume what 1000s of people want, but it sounds like a basic data science question rather than anything to do with structuring DS projects (e.g. using DVC).

So maybe changes like this will bring more traffic but also up the bounce rate. We'll have to try and see, I guess!

jeremydesroches replied:
> Does it matter that probably users looking for "data science use cases" are not looking for DVC use cases?
It's true that the term is not a perfect match, but it is related to the primary subject area (data science). Most non-brand terms are going to be partially related but inexact, as searches for discovery are imprecise (because they don't know what DVC is yet). 

The search engine is trying to fill in the gaps, so we want to expand on terms that are showing interest within the correct subject area in order to meet them halfway. This article already has some impressions for "use cases", including ML and data science so that's the motivation for this change.

jorgeorpinel replied:
OK, cool!

jorgeorpinel commented on content/docs/use-cases/index.md:
```diff
@@ -1,28 +1,28 @@
-please see our [Get Started](/doc/tutorials/get-started) instead.
+We provide short articles on common ML workflow and data science use cases that
+DVC can help with or improve. Our use cases are not written to be run end-to-end
+like tutorials. For more general, hands-on experience with DVC, please see our
```
```suggestion
like tutorials. For more general, hands-on experience with DVC, please see
```

jorgeorpinel commented on content/docs/use-cases/index.md:
```diff
@@ -1,28 +1,28 @@
-still lacks good practices for organizing their projects and collaborating
-effectively. This is a critical challenge: while ML algorithms and methods are
-no longer tribal knowledge, they are still difficult to implement, reuse, and
-manage.
+with deep learning and its applications in business, data scientists still lack
+good tools for organizing their projects and collaborating effectively. This is
+a critical challenge: while ML algorithms and methods are no longer tribal
```
Same Q. Is this SEO related? I'm trying to make sure the PR is well scoped. Looks like copy edits to me, and TBH I'm not very convinced:

- the data science community -> data scientists ?
- practices -> tools ?

jeremydesroches replied:
Yes, also SEO related: trying to associate "data science/scientists" with the terms "tools" and "project organization". I adjusted the original sentence to be more direct by referring to data scientists (specific role instead of general community) and tools (DVC as a tool as opposed to general practice.)

jorgeorpinel replied:
I see. Agree about the first change then ("data science community" doesn't seem like a query that gives us any traffic rn anyway) but still not sure about removing "good practices". Well (after checking in the SC), in fact we should change it to "best practices", but that term has potential of it's own I think.

Also speaking of tools here changes the semantics quite a bit, as we're trying to explain part of the motivation for DVC, and that has to do with DS best practices.

So can we change or add text somewhere else to associate "data scientists" and "tools"?

jeremydesroches replied:
Nice! This is even better. You are right about "best practices" — it's definitely another term we should be targeting. Added "tools" in the final bullet. That's a lightweight change and I think it's a better usage, keeping with how DVC is described elsewhere in the docs. Do you agree?

jorgeorpinel replied:
Agreed 🤝

jorgeorpinel commented on content/docs/use-cases/index.md:
```diff
@@ -1,28 +1,28 @@
-- understand how data or models were built in the first place;
-- be able to compare models and metrics to each other;
+- track, control, and switch between different versions of data or models
+  easily;
+- understand how machine learning models were built in the first place;
+- compare ML models and metrics to each other;
```
- OK with introducing "control" if we need it near "version", but if you want you can also add "version control" somewhere else in the doc or even in a new sentence if needed (not sure where).
- "machine learning models" 👍 (why not ML though? Up to you) but why was "data or" removed? The data part is also important, probably more that the models part 🙂

jeremydesroches replied:
> * OK with introducing "control" if we need it near "version", but if you want you could just add "version control" somewhere else in the doc instead, or even in a new sentence if needed (not sure where).
OK to have within proximity. 
> * "machine learning models" 👍 (why not ML though? Up to you) but why was "data or" removed? The data part is also important, probably more that the models part 🙂
The next bullet has ML, and we should have both. I thought the "data or" might be a typo because "...how data or models were built in the first place" implies "how data was built". Is that what we're trying to say or should it be "datasets or models"?

jorgeorpinel replied:
It's not a typo. We have "data and (ML) models" repeatedly throughout docs. Not sure what's wrong about "data was built". "Datasets" may be more correct in some ways but we want to have the term "data" a lot. Anyway, not something within the scope of this PR to decide on, let's just revert that part please.

jeremydesroches replied:
OK, understood. Updated to revert and adjust ML/machine learning placements. PTAL

jorgeorpinel replied:
Thanks.

jorgeorpinel replied:
BTW don't get me wrong, no problem in suggesting any changes you feel are necessary (just try to respect the scope more or less). I/we can always just ask to revert but that's normal 🙂

jorgeorpinel reviewed (commented):
Review of use cases index. Checking PR scope. Please generalize this feedback to the other docs before I get to review them. Thanks!

jorgeorpinel commented on content/docs/use-cases/versioning-data-and-model-files/index.md:
```diff
@@ -30,9 +30,9 @@ on-premises storage (e.g. SSH, NAS) as well as any major cloud storage provider
-data files and directories (among other purposes). They point to specific data
-contents in the <abbr>cache</abbr>, providing the ability to store multiple data
-versions out-of-the-box.
+the version of data files and directories (among other purposes). They point to
```
OK I see you're going for "versioning data files" here — thanks for adding the summary of terms in the PR description.

In DVC we try to make an important distinction between tracking and versioning though (the latter mostly left to Git) so let's keep both? May be as simple as

```suggestion
data files and directories for versioning (among other purposes). They point to
```

Or something else (again, don't be afraid to add a sentence somewhere if that doesn't work).

jorgeorpinel reviewed (changes requested):
Finished reviewing all the changes:

jeremydesroches commented on content/docs/use-cases/versioning-data-and-model-files/tutorial.md:
```diff
@@ -1,8 +1,8 @@
-machine learning version control scenario: working with multiple versions of
-datasets and ML models using DVC commands. We'll work with a
+machine learning version control scenario: managing multiple dataset and ML
+model versions using DVC commands. We'll work with a
```
Thanks! Made the change.

jeremydesroches commented on content/docs/use-cases/versioning-data-and-model-files/tutorial.md:
```diff
@@ -237,9 +237,9 @@ $ git commit -m "Second model, trained with 2000 images"
-and the DVC-files that point to them committed with Git. Let's now look at how
-DVC can help us go back to the previous version if we need to.
+That's it! We've tracked the second version of the dataset, model, and metrics
+in DVC and committed the DVC-files that point to them with Git. Now let's look
```
Thanks! "a second version" is better. Made the change.

Reverted to "Let's now" — I tried some other versions of the second sentence when I rewrote the first one, and unintentionally switched them. (IMO "Let's now" is a better usage for following a result with a new instruction. 😉 )

jorgeorpinel replied:
You may be right. Idk

https://english.stackexchange.com/questions/407003/now-lets-vs-lets-now-what-is-the-proper-sequence

jorgeorpinel commented:
@shcheklein would you like to review this first SEO PR too?

shcheklein commented:
@jeremydesroches @jorgeorpinel a lot of great improvements, guys! thanks!

jeremydesroches commented:
> BTW, you should have access to push branches to this repo @jeremydesroches so no need to use a fork going forward 🙂. Using branches directly on `origin` will trigger a review app that we can all see online, so it's preferred. Thanks

Hey @jorgeorpinel, I tried this earlier and got a permission error. Can you check if I have access? Thanks

jorgeorpinel commented:
Sure, we'll check.

jeremydesroches commented:
These four pages were submitted for reindexing and added to a list of docs to track for ongoing SEO performance.

jorgeorpinel commented:
OK, checking back on this, let's see if we can find anything on the Search Console now (over 10 days after merge).

Starting with https://dvc.org/doc/user-guide/what-is-dvc,

- [ ] firstly I discovered that there's still [some old URLs](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=page) that used to be under that one which are returning 404s... Should we setup a redirect even when it's pretty late?
- [ ] Looking at the [specific page only](https://search.google.com/u/1/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=page&metrics=CLICKS%2CIMPRESSIONS%2CCTR%2CPOSITION), there's no obvious change in the general terms (also very hard to tell with the week period — any way to smooth it out on a 7-day window here or on some other tool @jeremydesroches ? GA has that option for ex.)
  looking at the "machine learning" [terms](https://search.google.com/u/1/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=query&metrics=CLICKS%2CIMPRESSIONS%2CCTR%2CPOSITION&query=*machine%20learning) there's not much impact, but at least now it comes up in searches now and then 😋 
  ![image](https://user-images.githubusercontent.com/1477535/96517981-7fb0fb00-122f-11eb-8bcb-4a8e5e0cd614.png)

@jeremydesroches could you please look at the other 3 pages and report whether there is any impact so far/ share links to check back later on? Any other comments/tips on this evaluation process are appreciated. Thanks!

jeremydesroches commented:
> firstly I discovered that there's still some old URLs that used to be under that one which are returning 404s... Should we setup a redirect even when it's pretty late?

Hi @jorgeorpinel. No, a redirect won't do anything at this point because the old URLs aren't in the index anymore. You can check this for any given URL by clicking the magnifying glass (or searching for it in the bar at top on SC). 

> could you please look at the other 3 pages and report whether there is any impact so far/ share links to check back later on? Any other comments/tips on this evaluation process are appreciated. 

My assessment for each of the four pages is included below. The process I use is as follows:

1. set reasonable comparison period (in this case time since merge vs. previous period of same length)
2. review comparison curves/amounts for differences in clicks, impressions, CTR, average position
3. where there is a difference, switch to queries table and sort by most clicks or impressions
4. in descending order, look for notable click/impression/CTR/position gains or losses by term
5. when looking at a single metric, sort by difference in descending order to see terms with largest change
6. note patterns of change for target terms and new patterns/terms that appear (if any)
7. look at the 6 month or longer curve to see if any change in long-term trends

Sorry, no way to smooth things out to weekly in SC. For a comparison period this short it helps (me, anyway) to see the peaks/valleys. The curves are nice but in the end the aggregate and term-level gain/loss are the key metrics for any given period.

**What is DVC?** – [Merge - Nov 2 vs. previous period](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=query&metrics=IMPRESSIONS%2CPOSITION&start_date=20201008&end_date=20201102&compare_start_date=20200912&compare_end_date=20201007)
Incrementally more clicks and impressions. Improved and more consistent average position. Additional clicks/impressions and improved position on "mlflow"-related terms. Looking at a [longer period](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=query&metrics=POSITION&num_of_months=6) position is slightly higher and more consistent after the update. As you pointed out, the page started appear for new terms as well (even more now if you check your link again). Overall, a good example of how even small updates can spur changes to the index and across many pages add up to significant results.

**Use Cases Index** – [Merge - Nov 2 vs. previous period](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases&breakdown=query&metrics=IMPRESSIONS%2CPOSITION&start_date=20201008&end_date=20201102&compare_start_date=20200912&compare_end_date=20201007)
Big jump in impressions (3.5x). Good example of page being "auditioned" by Google to see where it should fall in index. The bump is likely to be temporary if there aren't more clicks or further changes. Most of the new impressions were around "use cases", notably "data science use cases" (the target term for the changes) and "ml use cases". The increased impressions show there is interest and it's likely we could expand further to potentially improve clicks and position.

**Versioning Data and Model Files Index** – [Merge - Nov 2 vs. previous period](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases%2Fversioning-data-and-model-files&breakdown=page&metrics=CLICKS%2CIMPRESSIONS%2CPOSITION&start_date=20201008&end_date=20201102&compare_start_date=20200912&compare_end_date=20201007)
Not a lot of overall movement here, but an improvement in average position is always good. There were 9 added clicks (10% increase) from the "model versioning" target term, so that worked. Expect we will have a lot more to talk about here when your latest changes are merged. Updates and improvements are good for search results!

**Versioning Data and Model Files Tutorial** – [Merge - Nov 2 vs. previous period](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases%2Fversioning-data-and-model-files%2Ftutorial&breakdown=query&metrics=IMPRESSIONS%2CPOSITION&start_date=20201008&end_date=20201102&compare_start_date=20200912&compare_end_date=20201007)
Slight growth in clicks and impressions. A few additional clicks from target terms "model versioning" and "dataset versioning", but nothing groundbreaking (yet!). Growth in impressions for target terms "data versioning" and "dataset versioning" too. Most significant here is an upward trend in average position over the period but also against the [long term trend](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases%2Fversioning-data-and-model-files%2Ftutorial&breakdown=query&metrics=POSITION&num_of_months=6). This is already a fairly popular page that produces a decent number of clicks, so only slight movement is expected from minor changes. Further updates could continue the uptrend — recommend a combination of content updates and term expansion.

jorgeorpinel commented:
OK I'll take a look at your specific reports ASAP. for now I just wanted to follow up on my own mini report as actually now there is more of an impact both in [what-is-dvc](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=page&metrics=CLICKS%2CIMPRESSIONS%2CCTR%2CPOSITION) which seems to have an improved avg position consistently:

![image](https://user-images.githubusercontent.com/1477535/98161973-3ce55900-1ea6-11eb-9750-c0acfaa75646.png)

But especially in the ["machine leaning"](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=!https%3A%2F%2Fdvc.org%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=query&metrics=CLICKS%2CIMPRESSIONS%2CCTR%2CPOSITION&query=*machine%20learning) term, which is starting to come alive:

![image](https://user-images.githubusercontent.com/1477535/98162043-5c7c8180-1ea6-11eb-90a1-7b912449717f.png)

So good work with those even if the change is small so far. At least we know there is a measurable impact!

jorgeorpinel commented:
Hi. Doing a final check on this, the first PR. Here are the results I can see:

My 2 previous mini-reports continued the same: improvements in the secondary? metrics (impressions and position).

For the other docs I just checked the weeks from the time they were merged vs. the same number of weeks (6) before that:

**What is DVC?** - [link](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*%2Fdoc%2Fuser-guide%2Fwhat-is-dvc&breakdown=query&metrics=IMPRESSIONS%2CCTR&start_date=20200827&end_date=20201008&compare_start_date=20201008&compare_end_date=20201119) - more impressions and better position, yet no more clicks. Maybe people just aren't that interested on this topic.

**Use Cases Index** - [link](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases&breakdown=query&metrics=CLICKS%2CIMPRESSIONS&start_date=20200827&end_date=20201008&compare_start_date=20201008&compare_end_date=20201119) - This one actually has ~10% more impressions AND ~15% more clicks, so the CTR improved 👍 . It suggests to me that this is one of the sections with the most potential for growth.

**Versioning Data and Model Files** use case - [link](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases%2Fversioning-data-and-model-files&breakdown=query&metrics=CLICKS%2CIMPRESSIONS&start_date=20200827&end_date=20201008&compare_start_date=20201008&compare_end_date=20201119) - similar as the previous one (even better clicks and CTR improvement) 
👍 but the position actually went down a little 🤷 — hopefully it will improve even more once the new versioning use case is released (almost...)

**Versioning Tutorial** - [link](https://search.google.com/u/0/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&page=*https%3A%2F%2Fdvc.org%2Fdoc%2Fuse-cases%2Fversioning-data-and-model-files%2Ftutorial&breakdown=query&metrics=CLICKS%2CIMPRESSIONS&start_date=20200827&end_date=20201008&compare_start_date=20201008&compare_end_date=20201119) - also good improvements (around 25% in clicks, for example)

So in general these are good results cc @shcheklein 

@jeremydesroches please start posting similar reports in your other PRs next week. Thanks

jeremydesroches commented:
> @jeremydesroches please start posting similar reports in your other PRs next week.

Thanks for checking on this @jorgeorpinel. Yes, I’ll do the same for other PRs this week along with the other GSoD reports.

jeremydesroches commented:
OK @jorgeorpinel, I've updated the other GSoD PRs with images now too. Please check my notes above as I added some images and updates to your original findings for this PR.
